### DISCUSSION TITLE
Help me with internatiolization for project

### DISCUSSION THREAD

**Discussion by nicitaacom** [i18next/i18next#2056] [Q&A] [answered]
### Step 1
git clone - https://github.com/nicitaacom/acc2-i18n

### Step 2
`pnpm i`
`pnpm dev`

![image](https://github.com/i18next/i18next/assets/39565703/7598d5bd-985c-42d5-ad12-0ef3dd3c19dd)

If you know how to male internatiolization using Next + TypeScript -
Please send example
or
Make PR

To make PR
1. Fork this repository
2. Choose 'for contribution option'
3. Make your changes and make it actually work
4. Create PR

Thank you!

**Comment by jamuhl** [comment, marked as answer]
what about heading to https://github.com/i18next/next-i18next and learn to do it?

**Reply by nicitaacom** [reply]
Yes thank you - I asked there and maintainer (as I understood owner) ansered me that its better reference to https://github.com/i18next/next-13-app-dir-i18next-example-ts

If you know how fix error that I got during build - please let me know here - https://github.com/i18next/next-i18next/discussions/2223#discussioncomment-7349759

**Reply by jamuhl** [reply]
as I don't use next.js at all - I got no idea - sorry

**Reply by nicitaacom** [reply]
That's fine
Thank you for replying for this discussing
